modules/virt-assigning-pci-device-virtual-machine.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-configuring-pci-passthrough.adoc
+
+[id="virt-assigning-pci-device-virtual-machine_{context}"]
+= Assigning a PCI device to a virtual machine
```

Comment:
This should be in a section "Configuring VMs" or so

Reply:
After fixing some structural items in the current update for this PR, will look into positioning the content as noted.